source/renderer/app/components/wallet/WalletSendForm.spec.tsx

```diff
@@ -131,7 +130,7 @@ describe('wallet/Wallet Send Form', () => {
     return async () => {
       fireEvent.mouseEnter(token);
       const removeTokenButton = await screen.findByTestId(
-        `remove-asset-${assets[tokenIndex].uniqueId}`
+        `remove-asset-${uniqueId}`
```

Comment:
Just a suggestion: This is the only data test id attribute written in kebab-case in our code base. Maybe it is worth to change it to the camelCase as you already change this part of code.